apps/src/StudioApp.js

```diff
@@ -251,6 +251,15 @@ StudioApp.prototype.init = function(config) {
     config = {};
   }
 
+  if (config.isLegacyShare && config.controllerAction == 'show') {
```

Comment:
I don't really like the idea of apps needing to know the name of a dashboard controller action. I wonder if we can just come up with a better name here?